src/gcp_scanner/crawler/bucket_names_crawler.py

```diff
@@ -0,0 +1,135 @@
+#  Copyright 2023 Google LLC
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+import json
+import logging
+import sys
+from typing import List, Dict, Any, Union, Tuple, TextIO
+
+from googleapiclient import discovery, errors
+
+from gcp_scanner.crawler.interface_crawler import ICrawler
+
+
+class BucketNamesCrawler(ICrawler):
```

Comment:
Ok, I am a bit confused :) you have bucket_names_crawler and storage_buckets_crawler. I think this file was added by accident or am I missing something?

Reply:
@mshudrak Thanks :rocket: I did not notice this one :confused: Updated now